ipn/ipnlocal/local.go

```diff
@@ -4053,6 +4058,7 @@ func (b *LocalBackend) resetForProfileChangeLockedOnEntry() error {
 	b.setNetMapLocked(nil) // Reset netmap.
 	// Reset the NetworkMap in the engine
 	b.e.SetNetworkMap(new(netmap.NetworkMap))
+	b.resetControlClientLocked()
```

Comment:
Is this necessary? We end up calling `resetControlClientLocked` in `Start` either way.

Reply:
Good point. removed.